server/lease/metrics.go

```diff
@@ -49,11 +49,19 @@ var (
 			// 1 second -> 3 months
 			Buckets: prometheus.ExponentialBuckets(1, 2, 24),
 		})
+
+	leaseTotal = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: "etcd_debugging",
+		Subsystem: "lease",
+		Name:      "total",
```

Comment:
I‘m wondering if `active_total`  would be a more suitable name for this metric. 
Cause i feel like this description is more precise.